tests/docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+version: '3'
+services:
+  graph-node:
+    image: graphprotocol/graph-node
+    userns_mode: "keep-id"
+    ports:
+      - '8000:8000'
+      - '8001:8001'
+      - '8020:8020'
+      - '8030:8030'
+      - '8040:8040'
+    depends_on:
+      - ipfs
+      - postgres
+    environment:
+      postgres_host: postgres
+      postgres_user: graph-node
+      postgres_pass: let-me-in
+      postgres_db: graph-node
+      ipfs: 'ipfs:5001'
+      ethereum: 'mainnet:http://anvil:8545'
+      GRAPH_LOG: info
+  ipfs:
+    image: ipfs/go-ipfs:v0.10.0
+    userns_mode: "keep-id"
+    ports:
+      - '5001:5001'
+  postgres:
+    image: postgres:14
+    userns_mode: "keep-id"
```

Comment:
This line should be removed (for all the containers, but particularly this one), as it was useful only when a local volume belonging to the current user id was mounted into the container.
Now that the container writes into its own directories (root), using the current unprivileged user causes denied access errors.